src/Setup/WixBA/WixBA.cs

```diff
@@ -61,6 +63,65 @@ public static void LaunchUrl(string uri)
         }
 
         /// <summary>
+        /// Open a log file.
+        /// </summary>
+        /// <param name="uri">URI to a log file.</param>
+        internal static void OpenLog(Uri uri)
+        {
+            // Switch the wait cursor since shellexec can take a second or so.
+            System.Windows.Input.Cursor cursor = WixBA.View.Cursor;
```

Comment:
Understood. Will do.  Procedurally I did not view this PR as a 'complete' feat5352 but as you requested that I submit 'functional' code so that others could try it out and we could do more refactoring.  I already started simplifying some of the XAML which has been my primary focus rather than the backing VM code.  I also found a VS extension  XamlStyler.Package.vsix) which I plan to try out as I know that I use a flat XAML syle different from what I assume most folks prefer.  So my procedure question is do I just make these changes and submit another PR or how do you want the PR and these changes integrated?